<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dataProcessing</class>
 <widget class="QMainWindow" name="dataProcessing">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dataProcessing</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="TableList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>511</width>
      <height>361</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>序号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>频率（GHz）</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>温度（K）</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>介电常数</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>介电损耗</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>0</string>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>0</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>498</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="OpenButton">
       <property name="text">
        <string>打开文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dataSetButton">
       <property name="text">
        <string>参数设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="calculateButton">
       <property name="text">
        <string>计算</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SaveDataButton">
       <property name="text">
        <string>保存数据</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>绘制 f-E</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>绘制 f-tan(E)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>绘制 T-E</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>绘制 T-tan(E)</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCalculate">
   <property name="text">
    <string>Calculate</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionFrequency_Epsilon">
   <property name="text">
    <string>Frequency_Epsilon</string>
   </property>
  </action>
  <action name="actionFrequency_TAN_Epsilon">
   <property name="text">
    <string>Frequency_TAN(Epsilon)</string>
   </property>
  </action>
  <action name="actionTemperature_Epsilon">
   <property name="text">
    <string>Temperature_Epsilon</string>
   </property>
  </action>
  <action name="actionTemperature_TAN_Epsilon">
   <property name="text">
    <string>Temperature_TAN(Epsilon)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
